### PR TITLE
Add stars

### PR DESCRIPTION
**Pull request by DenisKent**
Added real star component to the mealList and mealPages

### CODE REVIEW

File: client/src/pages/MealPage/sections/MealCookInfo/MealCookInfo.js [hunk 1]
```diff
@@ -22,15 +24,20 @@ class MealCookInfo extends Component {
       cook_firstname,
       cook_summary,
       cook_description,
-      cook_image_url
+      cook_image_url,
+      av_reviews,
+      count_reviews
     } = this.props.mealCookInfo.data;
+    const rating = Number(av_reviews)*10
```
Comment (shiryz) [inline client/src/pages/MealPage/sections/MealCookInfo/MealCookInfo.js]: same

File: client/src/pages/MealPage/sections/MealCookInfo/MealCookInfo.js [hunk 2]
```diff
@@ -49,7 +56,10 @@ MealCookInfo.propTypes = {
       cook_lastname: PropTypes.string,
       cook_summary: PropTypes.string,
       cook_description: PropTypes.string,
-      is_thumbnail: PropTypes.arr
+      cook_image_url: PropTypes.string,
+      is_thumbnail: PropTypes.arr,
+      av_reviews: PropTypes.string,
```
Comment (shiryz) [inline client/src/pages/MealPage/sections/MealCookInfo/MealCookInfo.js]: same

File: client/src/pages/MealsListPage/MealsListPage.js [hunk 3]
```diff
@@ -31,6 +31,7 @@ class MealListPage extends Component {
             const deadline = timeRemaining(
               new Date(meal.final_booking_at) - Date.now()
             );
+            const rating = Number(meal.av_star_rating)*10
```
Comment (shiryz) [inline client/src/pages/MealsListPage/MealsListPage.js]: same

File: client/src/pages/MealsListPage/MealsListPage.js [hunk 4]
```diff
@@ -81,7 +82,8 @@ const mealPropType = PropTypes.shape({
   count_review: PropTypes.string,
   price: PropTypes.string,
   final_booking_at: PropTypes.string,
-  meal_image_url: PropTypes.arr
+  meal_image_url: PropTypes.arr,
+  av_star_rating: PropTypes.string
```
Comment (shiryz) [inline client/src/pages/MealsListPage/MealsListPage.js]: same

File: server/database/getQueries/getMealCookInfo.js [hunk 5]
```diff
@@ -17,7 +17,7 @@ const getMealCookInfo = (mealId) => {
   LEFT JOIN
     (
       SELECT
-        reviews.cook_user_id, COUNT(reviews.star_rating) AS count_reviews
+        reviews.cook_user_id, COUNT(reviews.star_rating) AS count_reviews, AVG(reviews.star_rating) AS av_reviews
```
Comment (shiryz) [inline server/database/getQueries/getMealCookInfo.js]: same

File: testing/testPresetData.js [hunk 6]
```diff
@@ -55,6 +55,7 @@ const cookInfoResult = {
   cook_summary: 'Delicious recipies passed down 3 generations of Afifi',
   cook_image_url: 'https://user-images.githubusercontent.com/28222381/45268630-08251500-b488-11e8-94c4-7e664f5d45c9.jpg',
   count_reviews: '3',
+  av_reviews: '4.0000000000000000',
```
Comment (shiryz) [inline testing/testPresetData.js]: same

Comment (shiryz) [reply testing/testPresetData.js]: Where is this being tested?

Comment (matthall8) [reply testing/testPresetData.js]: It's been used as part of testing the query.

File: client/src/pages/MealPage/sections/MealCookInfo/MealCookInfo.js (hunk 2 above)
Comment (matthall8) [reply client/src/pages/MealPage/sections/MealCookInfo/MealCookInfo.js]: This cannot be changed without changing the name in the db which we agreed we wouldn't do.

**Pull request merged**